.github/workflows/ci.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           PR_GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
           PR_GITHUB_NAMESPACE: "checkmarx"
           PR_GITHUB_REPO_NAME: "ast-cli"
-          PR_GITHUB_NUMBER: 43
+          PR_GITHUB_NUMBER: 983
```

Comment:
what is this parameter?

Reply:
this is an old PR that we used for our PR decoration integration tests
it seems that this PR became to large (probably because of our decoration comments)
this is the reason for the failures that we had as a blocker in the last few days
I just replaced it by a new PR from today